app/models/ems_refresh/save_inventory.rb

```diff
@@ -1,14 +1,20 @@
 module EmsRefresh::SaveInventory
   def save_ems_inventory(ems, hashes, target = nil)
+    log_header = "EMS: [#{ems.name}], id: [#{ems.id}]"
+
     if hashes.kind_of?(Array)
+      _log.debug "#{log_header} Saving inventory using ManagerRefresh having received an object of array kind."
```

Comment:
Our logs already feel too big and seem to have a bunch of unnecessary information.
So keep if this is useful in production to diagnose problems
Remove if this was helpful writing the code but won't be much use on a customer's site.